Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,84 @@
-'use strict';
+"use strict";
 
+// 2.2
 const monday = [
   {
-    name: 'Write a summary HTML/CSS',
+    name: "Write a summary HTML/CSS",
     duration: 180
   },
   {
-    name: 'Some web development',
+    name: "Some web development",
     duration: 120
   },
   {
-    name: 'Fix homework for class10',
+    name: "Fix homework for class10",
     duration: 20
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   }
 ];
 
 const tuesday = [
   {
-    name: 'Keep writing summary',
+    name: "Keep writing summary",
     duration: 240
   },
   {
-    name: 'Some more web development',
+    name: "Some more web development",
     duration: 180
   },
   {
-    name: 'Staring out the window',
+    name: "Staring out the window",
     duration: 10
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   },
   {
-    name: 'Look at application assignments new students',
+    name: "Look at application assignments new students",
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+let durationOfTasks = tasks.map(Element => Element.duration);
```

Comment:
Since you are mapping the elements of a `tasks` array each element of that array is a `task`. So it would be more explicit to call the function argument `task` instead of the more generic `Element`. Moreover, variable, function and parameters names should be in camelCase (i.e. start with a lowercase letter).

Since you are extracting the `duration` values from the `tasks` it is a bit redundant to call the result of this mapping `durationOfTasks`. A simple `durations` will do fine. This name also implies that it is an array (because of the plural 's').

VSCode (ESLint) give a warning that the `durationOfTasks` variable is not re-assigned and therefore a `const` would be preferred over `let`. Do you get such warnings too? If not, check out the [VSCode Tips](https://github.com/HackYourFuture/fundamentals/tree/master/VSCodeTips#installation-instructions) to install ESLint. If you do get such warnings, please act on them. They help you to improve you code.

Reply:
Thanks for your preciouscommnets, i changed all things that you highlighted. I have already installed ESLint, but i did not get such a warning. I'll aperciate you if you check my VSCode tomorrow. I will come to you. 👍